src/devices/acurite.c

```diff
@@ -186,7 +190,7 @@ static int acurite_th_decode(r_device *decoder, bitbuffer_t *bitbuffer)
                 "id",               "",             DATA_INT,    id,
                 "battery",          "",             DATA_STRING, battery_low ? "LOW" : "OK",
                 "temperature_C",    "Temperature",  DATA_FORMAT, "%.1f C", DATA_DOUBLE, tempc,
-                "humidity",         "Humidity",     DATA_INT,    humidity,
+                "humidity",         "Humidity",     DATA_FORMAT, "%u %%", DATA_INT,    humidity,
```

Comment:
Why is this part of the the atlas change?   Will this potentially break parsers for other devices?

If all the humidity formatting it changed. It would be easier to find what changed/why if this was broken out into a separate PR